### DISCUSSION TITLE
A list of Integrated system with Meshcentral

### DISCUSSION THREAD

**Discussion by The-Real-Admin-Guy** [Ylianst/MeshCentral#4673] [Ideas]
Hi,
Is there a list of integrated systems?
What I mean is system like the Eset, BitDefender or Malwarebytes frameworks are often integrated within commercial systems.  Even TeamViewer has Malwarebytes integrated.

Backup Integration would be a big one.

Anyone did any work in that direction?
Anything planned?

**Comment by silversword411** [comment]
https://ylianst.github.io/MeshCentral/meshcentral/plugins/
